torch_xla/csrc/xla_backend_impl.h

```diff
@@ -41,6 +41,6 @@ class XLAData : public torch::lazy::BackendData {
 
 torch::lazy::BackendImplInterface* GetXlaBackendImpl();
 
-void InitXlaBackend();
+bool InitXlaBackend();
```

Comment:
What's the purpose of returning `bool`. I don't seem to see `xla_backend_inited` used anywhere in this PR. Is it to enable static initialization of `InitXlaBackend`? @JackCaoG?

Reply:
it is to enable static initialization in cpp test.